lib/scala/kg/src/main/scala/models/kg/KgPath.scala

```diff
@@ -7,7 +7,7 @@ package models.kg
 final case class KgPath(datasource: String, id: String, path: List[String]) {
```

Comment:
Nit, no need to change anything: this abstraction is probably going to go away, it was a stopgap so you could start on path visualization. I also anticipated that paths would be somewhat independent of benchmarks, but they haven't been.

Reply:
Ok thanks for the heads up